rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/PackageNameConverter.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2015 Google, Inc.
```

Comment:
```suggestion
 * Copyright 2019 Google, Inc.
```

Reply:
I left that date in that file as technically I only moved it `.groovy -> .java` though enough lines changed that github didn't catch it.  (The other files didn't have headers to start so I just put 2019.)

Reply:
Oh I see I see!